admin/package-files/systemd/flocker-agent.service

```diff
@@ -1,5 +1,5 @@
 [Unit]
-Description=Flocker Node Agent
+Description=Flocker ZFS Agent
```

Comment:
It'd be clearer if this file was renamed to `flocker-zfs-agent.service` or `flocker-dataset-agent.service` but we can do that when we make the dataset agent backend pluggable.